AP-protocol.md

```diff
@@ -263,13 +263,13 @@ To reassemble fragments into a single packet, a client or server uses the fragme
 
 2. It first checks the `session_id` in the header.
 
-3. If it has not received a fragment with the same `session_id`, then it creates a vector big enough where to copy the data of the fragments.
+3. If it has not received a fragment with the same `session_id`, then it creates a vector (`Vec<u8>` with capacity of
+   `total_n_fragments` * 80) where to copy the
+   data of the fragments;
 
-4. The client would need to create a `vec<u8>` with capacity of `total_n_fragments` * 80.
+4. It would then copy `length` elements of the `data` array at the correct offset in the vector.
```

Comment:
the `length` elements of the `data` array is a bit confusing.

Maybe something like copy the N elements of the `data` array where N is the `length`. Or something, but I think it could be clearer.

Fix this and I'll merge it, because there is nothing to vote on (just using right names

Reply:
Ok if I'm the only one which find it not super clear at first read than is fine like this